test/registration_spec.rb

```diff
@@ -201,38 +201,48 @@
 
   describe "#get_updates_list" do
     let(:base_product) { { "name" => "base" } }
+    let(:installer_update_base_product) { { "name" => "self_update_id" } }
     let(:remote_product) { { "name" => "base" } }
     let(:updates) { ["http://updates.suse.com/sles12/"] }
     let(:suse_connect) { double("suse_connect") }
+    let(:self_update_id) { "self_update_id" }
```

Comment:
I'd suggest using a real value here, e.g. the current value is `SLES` (see https://github.com/yast/skelcd-control-leanos/blob/master/control/control.leanos.xml#L76).

Using `self_update_id` is confusing, it's like naming variable `variable`...